packages/dashboard-frontend/src/services/dashboard-backend-client/websocketClient.ts

```diff
@@ -39,21 +49,26 @@ export class WebsocketClient {
       this.websocketStream.close();
     }
     const websocketContext = `${prefix}/websocket`;
-    const location = new URL(window.location.href).origin.replace('http', 'ws') + websocketContext;
+    const origin = new URL(window.location.href).origin;
+    const location = origin.replace('http', 'ws') + websocketContext;
     this.websocketStream = new ReconnectingWebSocket(
       location,
       [], {
       connectionTimeout: 10000,
       maxRetries: 10
     });
-    this.websocketStream.addEventListener('open', event => {
+
+    this.websocketStream.addEventListener('open', () => {
+      this.onDidWebSocketStatusOpen(websocketContext);
       deferred.resolve();
     });
     this.websocketStream.addEventListener('error', event => {
+      console.log(`WebSocket client '${websocketContext}' Error: ${event.message}`);
+      this.onDidWebSocketStatusFailing(websocketContext);
```

Comment:
Since Dashboard Backend do not store subscription by client/session, it's needed to resubscribe after connection is reastablished ( with using newer DevWorkspaceList resource version ), otherwise as currently, we get the following:
![Screenshot_20210903_153525](https://user-images.githubusercontent.com/5887312/132006223-a49ff6a4-76fb-42be-9892-ae9aee4ff358.png)

Reply:
I have fixed it